sdk-setup/src/mb2

```diff
@@ -1737,7 +1737,7 @@ run_apply() {
             fi
             patch_file="$(sed -n "s/^Patch$p: \+\(.*\)/\1/p" "$TMP_SPEC")"
             echo "$message $patch_file" >&2
-            patch "${common_op[@]}" $op < "$(dirname "$OPT_SPEC")/$patch_file" >&2
+            patch --binary "${common_op[@]}" $op < "$(dirname "$OPT_SPEC")/$patch_file" >&2
```

Comment:
The option should be added to `common_op`. I would be more assured if we added it conditionally, e.g., only if `$OPT_SPEC` contains CRLF line endings.